tests/test_bundle.py

```diff
@@ -554,20 +563,76 @@ def test_make_source_manifest(self):
         # quarto_inspection=None,  # type: typing.Optional[typing.Dict[str, typing.Any]]
 
         # No optional parameters
-        manifest = make_source_manifest(AppModes.PYTHON_API, None, None, None, None)
+        manifest = make_source_manifest(AppModes.PYTHON_API, None, None, None)
         self.assertEqual(
             manifest,
             {"version": 1, "metadata": {"appmode": "python-api"}, "files": {}},
         )
 
         # include image parameter
-        manifest = make_source_manifest(AppModes.PYTHON_API, None, None, None, "rstudio/connect:bionic")
+        manifest = make_source_manifest(AppModes.PYTHON_API, None, None, None,
+                                        image="rstudio/connect:bionic")
         self.assertEqual(
             manifest,
             {
                 "version": 1,
                 "metadata": {"appmode": "python-api"},
-                "environment": {"image": "rstudio/connect:bionic"},
+                "environment": {
```

Comment:
Do we wanna include a test for one of these that uses the shorthand flag?

Reply:
I added a test case in `main_test.py` that covers the callback func. It's not exactly what you're asking for but it's pretty close without actually having to print/parse a manifest.

7e3b4c68139e26efdf9cc7ff201e17d9212cc183